var/spack/repos/builtin/packages/aomp/package.py

```diff
@@ -91,7 +91,7 @@ class Aomp(Package):
     homepage = tools_url + "/aomp"
     url = tools_url + "/aomp/archive/rocm-3.10.0.tar.gz"
 
-    maintainers = ['srekolam', 'arjun-raj-kuppala', 'estewart08']
+    maintainers = ['srekolam', 'renjithravindrankannath', 'estewart08']
```

Comment:
aomp is no longer  updated for newer rocm releases.

Reply:
Reverted arjun-raj-kuppala maintainer to in aomp